.github/workflows/check.yml

```diff
@@ -5,101 +5,106 @@ env:
 
 jobs:
   check:
-    name: Neqo Build and Test
-    runs-on: ubuntu-22.04
-    env:
-      RUSTFLAGS: -C link-arg=-fuse-ld=lld
+    name: Build & test
     strategy:
+      fail-fast: false
       matrix:
-        rust-toolchain: [stable, 1.65.0]
+        os: [ubuntu-latest, macos-latest] #, windows-latest]
+        rust-toolchain: [1.65.0, stable] #, beta]
+    runs-on: ${{ matrix.os }}
+    defaults:
+          run:
+            shell: ${{ matrix.os == 'windows-latest' && 'msys2 {0}' || 'bash' }}
 
     steps:
-      - name: Install Packages
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust-toolchain }}
+          components: rustfmt, clippy
+
+      - name: Install Rust tools
+        uses: taiki-e/install-action@v2
+        with:
+          tool: nextest, cargo-llvm-cov
```

Comment:
Does `cargo install X` work for these two?  It seems to.  And it might be easier than getting approval for the action.

Reply:
Yup, it occurred to me to that this action might not be needed. Will replace it.